site/topics/control-logic/instructions-microcodes.rst

```diff
@@ -184,6 +179,176 @@ Microcodes
 Fetch and Instruction Register
 ==============================
 
+* Remember, RAM stores both instructions and data
+* Continuing the above example of loading data from RAM to register A
+* Although the control logic to perform this action was discussed above, the instruction must have first come from RAM
+
+    * The instruction would need to be *fetched* from RAM
+
+
+* Instructions will be stored sequentially within RAM, starting at address 0
```

Comment:
Mentioned before, pointing out this is another instance of the content feeling like it is being told, rather than being discovered.